src/services/config/WorkspaceSchema.js

```diff
@@ -142,6 +142,23 @@ const basicWebAppOptions = z.object({
               description: LABELS_DICT,
               kpis: z.array(TWINGRAPH_INDICATOR).optional().nullable(),
               attributes: z.array(z.string().optional()).optional().nullable(),
+              previewTable: z
+                .object({
+                  columns: z
+                    .array(
+                      z
+                        .object({ field: z.string(), headerName: z.string(), type: z.array(z.string()) })
+                        .strict()
+                        .optional()
+                        .nullable()
+                    )
+                    .optional()
+                    .nullable(),
+                  queryId: z.string(),
```

Comment:
It's just a question, but why only `queryId` isn't `optional` neither `nullable` ? The component cannot be displayed without columns, so I'd say either both have to be `optional` and `nullable` or none of them :slightly_smiling_face: